.github/workflows/linter.yml

```diff
@@ -0,0 +1,127 @@
+name: Run Linter
+
+on:
+  pull_request:
+
+concurrency:
+  group: linter-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  DOCKER_LINTER_DEPENDENCY_IMAGE_NAME: ghcr.io/genspectrum/lapis-silo-linter-dependencies
+
+jobs:
+  linterDependencies:
+    name: Build linter dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          echo "Github Event ${{ github.event }}"
+          echo "Github Event number ${{ github.event.number }}"
+          echo "github.repository_owner ${{ github.repository_owner }}"
+          echo "github.repository ${{ github.repository }}"
+          echo "Changed files in PR #${PR_NUMBER}:"
+          $files=$(curl -s \
+           "https://api.github.com/repos/${{ github.repository }}/pulls/${PR_NUMBER}/files" \
+           | jq -r '.[].filename')
+          echo "${$files}"
+          echo "::set-output name=files::$files"
```

Comment:
Is this step actually necessary?

Reply:
Nope, it was just for testing. And I put it first in the file for quicker trial and error :D